openstack-seeder/python/openstack_seeder.py

```diff
@@ -2279,7 +2329,9 @@ def seed_config(config, args, sess):
 
     if 'domains' in config:
         for domain in config['domains']:
-            seed_domain(domain, args, sess)
```

Comment:
I believe this should remain as it was.. 
The evaluation of the need for ec2_cred seeding is done in line 729

Reply:
You're right.  That was to filter out the seeding to speed up debugging.  Think I fixed it.